usr/sbin/pkcsconf/pkcsconf.c

```diff
@@ -991,14 +979,11 @@ CK_RV init_token(int slot_id, CK_CHAR_PTR pin)
 
     rc = FunctionPtr->C_InitToken(slot_id, pin, pinlen, label);
     if (rc != CKR_OK) {
-        if (rc == CKR_PIN_INCORRECT) {
-            printf("Incorrect PIN Entered.\n");
-            fflush(stdout);
-        } else {
-            printf("Error initializing token: 0x%lX (%s)\n", rc,
-                   p11_get_ckr(rc));
-            fflush(stdout);
-        }
+        if (rc == CKR_PIN_INCORRECT)
+            warnx("Incorrect PIN Entered.");
+        else
```

Comment:
... entered (?)

Reply:
yes, fixed those, too.
No need to re-approve :-)